catroid/src/main/res/layout/activity_upload.xml

```diff
@@ -44,65 +44,69 @@
 
         <ImageView
             android:id="@+id/project_image_view"
-            android:layout_width="100dp"
-            android:layout_height="100dp"
+            android:layout_width="72dp"
+            android:layout_height="65dp"
+            android:layout_marginStart="@dimen/dialog_input_offset"
+            android:background="@drawable/image_contour"
+            android:cropToPadding="true"
+            android:scaleType="centerCrop"
             android:layout_marginBottom="@dimen/details_spacing"/>
 
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:orientation="vertical">
 
-        <LinearLayout
```

Comment:
this is misleading, no single change from line 56 on, please revert it and try to avoid Reformat Code (alt+ctrl+L) for the whole file, use it only for the new code.